src/frontend/src/flows/pin/pinInfo.json

```diff
@@ -0,0 +1,18 @@
+{
+  "en": {
+    "continue_and_set_pin": "Continue and set pin",
+    "cancel": "Cancel",
```

Comment:
Not for this PR but I can "Cancel" is redundant

Reply:
I mean that I bet that you have `"cancel": "Cancel"` in other i18n json files in the code base.

Reply:
Ah, yes. But I think this is intentional, so that the translation for cancel can depend on the context.
CC: @nmattia